app/javascript/sidebar-widget/sidebarUser.jsx

```diff
@@ -34,7 +34,7 @@ class SidebarUser extends Component {
             onClick={this.onClick}
             id={`widget-list-item__follow-button-${user.username}`}
           >
-            {user.following ? '✓ FOLLOWING' : '+ FOLLOW'}
+            {user.following ? 'Following' : 'Follow'}
```

Comment:
Do we still want to keep the checkmark as well for `Following`?

Reply:
Skipping checkmark and plus was actually intentional. ✅